### PR TITLE
Adding ATIP documentation structure

### PR DESCRIPTION
**Pull request by garymackenzie**
Adding structure for ATIP documentation and outline content

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -51,6 +51,18 @@ const sidebars = {
         }
       ]
     },
+    {
+      "type": "category",
+      "collapsible": true,
+      "collapsed": false,
+      "label": "Documentation",
```
Comment (rdoxenham) [inline sidebars.js:58]: Can we rename this to "Product Documentation"?

File: sidebars.js [hunk 2]
```diff
@@ -51,6 +51,18 @@ const sidebars = {
...
+      "items": [
+        {
+          type: 'autogenerated',
```
Comment (rdoxenham) [inline sidebars.js:61]: I think that "autogenerated" here means that we end up with "atip" in the menu bar. I'm wondering if it's possible to hard code this to "ATIP" or "Adaptive Telco Infrastructure Platform" so it looks a bit more professional?

File: docs/product/atip/requirements.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+sidebar_position: 3
+title: Pre-Requisites and Assumptions
```
Comment (rdoxenham) [inline docs/product/atip/requirements.md:3]: I think this is "Prerequisites"?

File: docs/product/atip/introduction.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 1
+title: Introduction to ATIP Documentation
```
Comment (rdoxenham) [inline docs/product/atip/introduction.md:3]: Think we can just leave this as "Introduction"

**Review by rdoxenham — APPROVED** [review]
LGTM, thanks Gary!

File: sidebars.js [hunk 5]
```diff
@@ -51,6 +51,26 @@ const sidebars = {
...
+    {
+      "type": "category",
+      "collapsible": true,
+      "collapsed": true,
+      "label": "Product Documentation",
+      "items": [
+        {
+          "type": "category",
+          "collapsible": true,
+          "collapsed": true,
```
Comment (e-minguez) [inline sidebars.js:63]: Just a nit but I think it would be better if this is false as it requires two clicks to see the actual content.

**Review by e-minguez — CHANGES_REQUESTED** [review]

**Comment by e-minguez** [comment]
LGTM but as it is incomplete, can we mark it as "DRAFT" or something like that?

File: sidebars.js (hunk 5 above)
Comment (garymackenzie) [reply sidebars.js:63]: I've made Product Documentation expanded and ATIP collapsed as discussed...

**Comment by garymackenzie** [comment]
> LGTM but as it is incomplete, can we mark it as "DRAFT" or something like that?

Yes, good call - category and all pages now marked as *DRAFT and note added to introduction

**Review by e-minguez — APPROVED** [review]

**Pull request merged**